### ISSUE TITLE
Stats

### ISSUE THREAD

**Issue by blank85** [vendetta-mod/ideas#71]
### Discord username

blan.k

### Plugin description

A plugin where we can see our account stats. Like how much messages we sent and much more

### Elaborate

It should be a command or In plugin settings 

### ​

- [X] I have verified that this request is indeed original

**Comment by solonovamax** [comment]
something like message counts would be difficult bc it'd require a *ton* of api requests, and discord will *not* like that

**Comment by blank85** [comment]
Well discord has severe skil issue

**Comment by Gabe616** [comment]
yeah discord surprisingly enough doesn't track how many messages you send, what a shocker

**Comment by blank85** [comment]
😭

**Comment by acquitelol** [comment]
i mean its possible but it would only work for the messages sent on your client where the plugin is installed and nowhere else, you can hook into a flux dispatch for sending a message or hook into sendMessage on the module where startEditEvent is or whatever and handle that
